curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md

```diff
@@ -25,14 +25,12 @@ function equalityTest(myVal) {
 If `myVal` is equal to `10`, the equality operator returns `true`, so the code in the curly braces will execute, and the function will return `Equal`. Otherwise, the function will return `Not Equal`. In order for JavaScript to compare two different <dfn>data types</dfn> (for example, `numbers` and `strings`), it must convert one type to another. This is known as Type Coercion. Once it does, however, it can compare terms as follows:
 
 ```js
-1   ==  1
-1   ==  2
-1   == '1'
-"3" ==  3
+1   ==  1 // true
+1   ==  2 // false
+1   == '1' // true
+"3" ==  3 // true
```

Comment:
```suggestion
1   ==  1  // true
1   ==  2  // false
1   == '1' // true
"3" ==  3  // true
```

Reply:
Yea for sure! should i just commit suggestion from here? sorry never used this feature.

Reply:
If you switch to the `files` tab, you can go through and hit "Add suggestion to batch" on each of them - that'll let you commit them all at once.

Reply:
thanks! have a good day